docs/earthly-command/earthly-command.md

```diff
@@ -783,3 +783,23 @@ $ earthly doc +build
 +build
   build runs 'go build' and saves the artifact locally.
 ```
+
+## earthly web
+
+#### Synopsis
+
+* ```
+  earthly web [--provider=<provider-ref>]]
+  ```
+
+#### Description
+
+Prints url for logging into the CI application and attempts to open your default browser with that url.
```

Comment:
can you expand on `logging into the ci`; I thought this command requires that you are already logged into earthly, and this passes along your authentication token to the ci -- essentially allowing you to skip the login portion of the CI.

Reply:
It doesn't require you to be logged in - but it will pass along your authentication token if you are.
Unless you specify the' provider', you are brought to a login page. 
When `provider` is specified, the CI will attempt a login to that provider automatically.

Reply:
We chatted in Slack and Alex helped with a clear description